components/FMenu.vue

```diff
@@ -419,7 +438,9 @@ function handlePreselectSearch(event: KeyboardEvent) {
 
   function matchResult(option: FMenuOption) {
     const optionLabel = removeDiacritics(option.label.toLowerCase());
-    const optionValue = removeDiacritics(('' + option.value).toLowerCase());
+    const optionValue = removeDiacritics(
+      ('' + stringify(option.value)).toLowerCase()
```

Comment:
you stringify function always returns a string, so you no-longer need to cast the option value into string with `'' + ...`
```suggestion
      stringify(option.value).toLowerCase()
```